apis/src/components/organisms/history.rs

```diff
@@ -179,7 +179,7 @@ pub fn History(#[prop(optional)] extend_tw_classes: &'static str) -> impl IntoVi
                 <Reserve alignment=Alignment::DoubleRow color=Color::White/>
                 <Reserve alignment=Alignment::DoubleRow color=Color::Black/>
             </div>
-            <div ref=parent class="grid overflow-auto grid-cols-4 gap-1 mb-8 max-h-full h-fit">
+            <div ref=parent class="grid grid-cols-4 gap-1 mb-8">
```

Comment:
These changes broke the history tabs scroll in an ugly way. you can compare by opening a long game on hivegame and trying to go backwards and forwards in it, using all four of the history navigation buttons. 
[game](https://hivegame.com/game/2IsfDdPT1r) and a long game locally in play.

Reverting fixes it, maybe one of the classes is superfluous but at least some of them are needed .

Before releasing this feature I'm going to add another pr on top that tries to improve some of the interface/style issues. So I can fix it in mine as well